Tasks/AzureRmWebAppDeploymentV3/azurermwebappdeployment.ts

```diff
@@ -0,0 +1,147 @@
+import tl = require('vsts-task-lib/task');
```

Comment:
Apart from copy of V3 task, can you call out the places where you have done changes so that it can work with latest common. It becomes easy to review the delta. Right now, because of copy, the diff is huge and the delta (if any) is not prominent.

Reply:
There is literally no delta other than the task.json file. I copy pasted the AzureRmWebAppDeployment folder from releases/m131 branch to the current master.